tasks/ubuntu_repositories.yml

```diff
@@ -1,6 +1,18 @@
 ---
 - name: Add Ubuntu repositories
+  become: true
```

Comment:
The role assumes it is being run as root so you can drop this

Reply:
Will do.